orcid-core/src/main/java/org/orcid/core/oauth/OrcidOauth2UserAuthentication.java

```diff
@@ -169,7 +169,7 @@ public String getName() {
             return StringUtils.EMPTY;
         }
         
-        if(Visibility.PUBLIC.value().equals(recordName.getVisibility())) {
```

Comment:
I'm not sure why `Visibility` has a `value()` method which is the same than its `toString()` one so maybe I'm missing something here